Cannot update to latest version: No compatible version found: collapse-whitespace

When running `npm update` I get the following error:
```bash
npm http 304 https://registry.npmjs.org/collapse-whitespace
npm ERR! Error: No compatible version found: collapse-whitespace@'domchristie/collapse-whitespace#numerical_node_type'
npm ERR! Valid install targets:
npm ERR! ["1.0.0","1.0.1","1.0.2","1.0.3","1.0.4","1.0.5","1.1.0","1.1.1"]
npm ERR!     at installTargetsError (/usr/share/npm/lib/cache.js:719:10)
npm ERR!     at /usr/share/npm/lib/cache.js:638:10
npm ERR!     at saved (/usr/share/npm/node_modules/npm-registry-client/lib/get.js:142:7)
npm ERR!     at /usr/lib/nodejs/graceful-fs/polyfills.js:133:7
npm ERR!     at Object.oncomplete (fs.js:107:15)
```

Is the version number correct for collapse-whitespace correct?

Thanks

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "jsdom": false
   },
   "dependencies": {
-    "collapse-whitespace": "domchristie/collapse-whitespace#numerical_node_type",
+    "collapse-whitespace": "1.1.1",
     "jsdom": "^3.1.2"
   }
 }

